### ISSUE TITLE
"/usr/share/kservices5/imagethumbnail.desktop" does not exist on KDE neon 5.26

### ISSUE THREAD

**Issue by Mar2ck** [novomesk/qt-jpegxl-image-plugin#23]
Following the tutorial I have the .so built and installed but when going to edit the `/usr/share/kservices5/imagethumbnail.desktop` file it seems it no longer exists. Apt-file says the package that should contain this file is kio-extras-data but the newest version of that package (22.08.2) no longer has it, atleast on KDE neon. 
So I'm at the point where jpegxl files will display in Gwenview and have thumbnails generated in Dolphin but only after changing the files extension to .webp, otherwise it refuses to load. Do you know an alternate way of registering the filetype?

OS Info:
```
Operating System: KDE neon 5.26
KDE Plasma Version: 5.26.2
KDE Frameworks Version: 5.99.0
Qt Version: 5.15.6
```

**Comment by novomesk** [comment]
If the kio-extras is new, it doesn't need to be modified.

Check the version of `shared-mime-info` package. If it is older than 2.2, you need to register `image/jxl` mime type by copying `image-jxl.xml` from libjxl source file into `/usr/share/mime/packages/` and running `update-mime-database /usr/share/mime/`

**Comment by Mar2ck** [comment]
`shared-mime-info` is 2.1-2 for me, the `image-jxl.xml` is in the mime packages folder and i've run the update command to no avail. 
I tried creating a jxl.desktop file in `/usr/share/kservices5/qimageioplugins/` but that didn't work either

**Comment by novomesk** [comment]
I tried KDE Neon in Virtualbox and I was successful.

1. Registered mime type
2. Restarted dolphin (clicking JXL file launches gwenview but with error)
3. Installed development tools and libraries
4. Built and installed libjxl 0.7.x
5. build_libqjpegxl_dynamic.sh and copied the libqjpegxl.so to the folder where other Qt plugin are installed.
6. After restart of dolphin I see thumbnail of test JXL.

**Comment by Mar2ck** [comment]
Turns out the issue was my mistake. Back before i tried using this plugin, i set .jxl files to open in gThumb by default. Since my system didn't have a `.jxl` mime at the time, KDE created one under `application/x-kdeuser1` which i only happened to figure out now by hovering over the type in Dolphin. 
![mime-type-before](https://user-images.githubusercontent.com/6759254/199360386-5355a15d-9d1a-4d20-91da-62340f9b11a1.png)
I fixed it by deleting that custom mime type in System Settings -> File Associations and then it just started worked.
![mime-type-after](https://user-images.githubusercontent.com/6759254/199360564-bb3750c2-4a20-42a5-a049-b7e417a3cb9b.png)
Thank you for trying to help me with this though, even though it ended up being my bad in the end lol
